eva-repository/src/testFixtures/kotlin/com/razz/eva/persistence/executor/FakeMemorizingQueryExecutor.kt

```diff
@@ -66,6 +66,14 @@ class FakeMemorizingQueryExecutor(
             .execute(jooqQuery.getSQL(INLINED))
     }
 
+    override suspend fun executeSetSession(
+        dslContext: DSLContext,
+        sessionParamName: String,
+        sessionParamValue: String,
+    ) {
+        return
+    }
```

Comment:
I think proper place is to override/extend `TransactionManager::withConnection` and `TransactionManager::inTransaction`. Also check usages of [PrimaryConnectionRequiredFlag](https://github.com/razz-team/eva/blob/2907bd6c27d23715d948697f84dfc91ee3be0adf/eva-persistence/src/main/kotlin/com/razz/eva/persistence/PrimaryConnectionRequiredFlag.kt) - this is an idiomatic way to pass _context_ in async (suspend) environment. So we can explore something like 
```kotlin

data class RowsOwners(val owners set<ModelId<out Comparable<*>>>) : CoroutineContext.Element
```